src/Microsoft.ML.AutoML/Utils/MLNetUtils/Conversions.cs

```diff
@@ -25,7 +26,7 @@ public static bool TryParse(in TX src, out R4 dst)
                 dst = R4.NaN;
                 return true;
             }
-            if (float.TryParse(str, out dst))
+            if (float.TryParse(str, NumberStyles.Float, CultureInfo.InvariantCulture, out dst))
```

Comment:
Any idea what the ML.NET TextLoader does? Does the TextLoader handle reading as `InvariantCulture` correctly?

I'm worried that infer the value is a number in the AutoML code, then the TextLoader won't be able to read the value since it will read as the local culture. If so, we'd likely want to also correct the TextLoader.